docs/changelog-fragments.d/866.bugfix.md

```diff
@@ -0,0 +1,5 @@
+Detect and error for missing `/dev/mqueue` directory when
+execution environment support is enabled and the container
+engine is set to `podman`. This is a condition commonly
+found using Ubuntu with WSL2.
+-- by {user}`cidrblock`
```

Comment:
IIRC Markdown would inline this into the paragraph in there's no empty line
```suggestion

-- by {user}`cidrblock`
```

Reply:
added, looks better